components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/statusview/StatusView.kt

```diff
@@ -56,7 +56,7 @@ object StatusView {
         bodyContentDesc: String = "",
         bodyAlignment: BodyAlignment = BodyAlignment.CENTER,
         textColor: Color = HmrcTheme.colors.hmrcBlack,
-        icon: Int,
+        icon: Int = R.drawable.ic_maintenance,
```

Comment:
I don't think we need to provide a default value here.. it's just a required field for all consumers. The status view can be used for many scenarios and we don't really want to link it to the maintenance status at the component level.